app/scripts/actions/projectActions.js

```diff
@@ -314,25 +311,8 @@ ProjectActions.editFile.preEmit = function (id, fileName) {
     });
 };
 
-ProjectActions.getContainer.preEmit = (id) => {
-    fetch(urlGen.routes.ddsUrl + 'folders/' + id, {
-        method: 'get',
-        headers: {
-            'Authorization': appConfig.apiToken,
-            'Accept': 'application/json'
-        }
-    }).then(checkResponse).then(function (response) {
-        return response.json()
-    }).then(function (json) {
-        ProjectActions.getContainerSuccess(json)
-    })
-        .catch(function (ex) {
-            ProjectActions.getContainerError(ex)
-        });
-};
-
-ProjectActions.getFileContainer.preEmit = (id) => {
-    fetch(urlGen.routes.ddsUrl + 'files/' + id, {
+ProjectActions.getEntity.preEmit = (id, kind) => {
+    fetch(urlGen.routes.ddsUrl + kind + id, {
```

Comment:
Kind in the API is dds-folder and dds-file, so this isn't intended to follow the standards set by the api. It's simply for use in this one situation. I will concatenate in the '/' though.